podkrepi-bg/frontend
# Add title to campaign page

Closes #988

## Motivation and context

The campaign title is not shown in the document title which leads to poor SEO and user experience.

## Screenshots:

![image](https://user-images.githubusercontent.com/893608/183074365-a8885645-6bc1-41aa-8c27-bd05b55146cf.png)


## Testing

1. Go to the campaign page of individual campaign
2. Check the title - should match the campaign title
